modules/characterSheetsLevelEligibility.json

```diff
@@ -0,0 +1,367 @@
+{
+    "name": "CharacterSheets Level",
+    "details": [
+      "A Hats Protocol eligibility module that checks players for CharacterSheets characters with a minimum level for specific classes."
+    ],
+    "links": [
+      {
+        "label": "GitHub",
+        "link": "https://github.com/raid-guild/character-sheets-contracts"
+      },
+      {
+        "label": "CharacterSheets",
+        "link": "https://charactersheets.io"
+      },
+      {
+        "label": "RaidGuild",
+        "link": "https://raidguild.org"
+      }
+    ],
+    "parameters": [
+      {
+        "label": "Class IDs",
+        "functionName": "classIds",
+        "displayType": "number[]"
+      },
+      {
+        "label": "Min Levels",
+        "functionName": "minLevels",
+        "displayType": "number[]"
+      },
+      {
+        "label": "CharacterSheet",
+        "functionName": "SHEETS_ADDRESS",
+        "displayType": "address"
+      },
+      {
+        "label": "Classes Token",
+        "functionName": "CLASSES_ADDRESS",
+        "displayType": "erc1155"
+      }
+    ],
+    "type": {
+      "eligibility": true,
+      "toggle": false,
+      "hatter": false
+    },
+    "implementationAddress": "0x937f17D42959c53F0ff22FfdBb22C04Fe1F09cF4",
+    "creationArgs": {
+      "useHatId": true,
+      "immutable": [
+        {
+          "name": "Classes Address",
+          "description": "The address for the Classes token used in the character sheets game",
+          "type": "address",
+          "example": "0xaddress",
```

Comment:
These examples should be real values. They are used in the testing scripts to verify that module instances can indeed be deployed as expected.